react-ui/src/common/FeatureBanner.jsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Notify users of new features
+ */
+
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import { useMemo } from 'react';
+
+/**
+ * Enum of the available screens.
+ * @typedef {'submission' | 'voting'} Screen
+ */
+const SCREENS = {
+  HOME: 'home',
+  SUBMISSION: 'submission',
+  VOTING: 'voting',
+};
+
+/**
+ * Array of features.
+ * @type {Feature[]}
+ * ------------
+ * A feature list for a particular month
+ * @typedef {Object} Feature
+ * @property {string} month_year - The month and year when the feature was introduced.
+ * @property {Bullet[]} bullets - An array of bullet points describing the feature details.
+ * ------------
+ * An individual feature
+ * @typedef {Object} Bullet
+ * @property {boolean} isBold - Indicates whether the bullet point should be displayed in bold.
+ * @property {string} text - The text content of the bullet point.
+ * @property {string[]} onScreens - An array of screens where the feature is applicable.
+ */
+const features = [
```

Comment:
Is this just a placeholder? Seems like this would be better suited for the database.

Reply:
Whichever is easiest. Database would certainly help enforce type, and maybe we could allow mod access if required at some point.